doc/development/deprecations.rst

```diff
@@ -9,6 +9,12 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+* The functions ``qml.qinfo.classical_fisher`` and ``qml.qinfo.quantum_fisher`` are deprecated. Instead,
+  ``qml.gradients.classical_fisher`` and ``qml.gradients.quantum_fisher`` should be used.
+
+  - Deprecated in v0.38
+  - Will be removed in v0.39
```

Comment:
I might phrase this as _moved_, since the functionality remains exactly the same (?)